python/fusion_engine_client/messages/control.py

```diff
@@ -176,41 +176,94 @@ class ResetRequest(MessagePayload):
     ##
     # Perform a device hot start.
     #
-    # A hot start is typically used to restart the navigation engine in a
-    # deterministic state, particularly for logging purposes.
+    # This will reset the navigation engine into a known state, using previously
+    # stored position and time information. The device will begin navigating
+    # immediately if possible.
     #
     # To be reset:
     # - The navigation engine (@ref RESTART_NAVIGATION_ENGINE)
-    # - All runtime data (GNSS corrections (@ref RESET_GNSS_CORRECTIONS), etc.)
     #
-    # Not reset:
+    # Not reset/performed:
+    # - All runtime data (GNSS corrections (@ref RESET_GNSS_CORRECTIONS), etc.)
     # - Position, velocity, orientation (@ref RESET_POSITION_DATA)
+    # - GNSS ephemeris data (@ref RESET_EPHEMERIS)
     # - Calibration data (@ref RESET_CALIBRATION_DATA)
     # - User configuration settings (@ref RESET_CONFIG)
-    # - GNSS measurement engine (@ref RESTART_GNSS_MEASUREMENT_ENGINE)
+    # - Reboot GNSS measurement engine (@ref RESTART_GNSS_MEASUREMENT_ENGINE)
```

Comment:
It seems below is the full list for Hot (missing "Fast IMU...", "Training prameters...", and "Reboot navigtion processor"), using the same 10 bulleted items in all the starting methods. (Same 10 items appear every time, more items move from bottom to top as you go down, is what I kind of expect.)
```
    # To be reset:
    # - The navigation engine (@ref RESTART_NAVIGATION_ENGINE)
    #
    # Not reset/performed:
    # - All runtime data (GNSS corrections (@ref RESET_GNSS_CORRECTIONS), etc.)
    # - Position, velocity, orientation (@ref RESET_POSITION_DATA)
    # - GNSS ephemeris data (@ref RESET_EPHEMERIS)
    # - Fast IMU corrections (@ref RESET_FAST_IMU_CORRECTIONS)
    # - Training parameters (slowly estimated IMU corrections, temperature
    #   compensation, etc.; @ref RESET_NAVIGATION_ENGINE_DATA)
    # - Calibration data (@ref RESET_CALIBRATION_DATA)
    # - User configuration settings (@ref RESET_CONFIG)
    # - Reboot GNSS measurement engine (@ref RESTART_GNSS_MEASUREMENT_ENGINE)
    # - Reboot navigation processor (@ref REBOOT_NAVIGATION_PROCESSOR)
```